### DISCUSSION TITLE
Taxes Report

### DISCUSSION THREAD

**Discussion by mbugikp** [Blair2004/NexoPOS#1450] [Ideas]
**Describe the bug**
When you want a taxes report, where can you find one. Most people apply more than one tax on a product. Therefore each tax should have its figures reported separately. There is a sales report with a section called sales taxes, but the amounts are combined we need to separate tax amounts for each tax unit. What about purchase tax? Do we have a report showing all purchase tax amounts?

**To Reproduce**

**Expected behavior**
A sales report showing each tax unit and amount separately, A purchase report showing each tax unit and amount separately, 

**Screenshots**
![taxes report](https://user-images.githubusercontent.com/13999977/225720685-e4b16a5d-a9ef-42f3-8935-65bda8d43f54.png)

**Environment:**
 - NexoPOS 4.8.15 
 - Premium Extensions: [e.g Nexo Print Adapter 4.8.5 Gastro 4.8.14

**Comment by Blair2004** [comment]
We agree on this one.
